network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/listing_block.html

```diff
@@ -20,6 +23,16 @@
                         <img src="{{ img.url }}" alt="{{ card.alt_text }}"/>
                         <div class="tw-flex tw-flex-1">
                             <div class="tw-relative tw-pt-8 tw-w-full tw-flex tw-flex-col">
+                                {% if card.category or card.date_meta %}
+                                    <div class="tw-row tw-px-8 tw-py-2">
+                                        {% if card.category %}
+                                            <div class="tw-px-0"><p class="tw-h6-heading tw-text-blue-80 tw-mr-2 tw-py-2 tw-mb-0">{{ card.category }}</p></div>
+                                        {% endif %}
+                                        {% if card.date_meta %}
+                                            <div class="tw-px-2"><p class="tw-h6-heading tw-text-gray-40 tw-py-2 tw-mb-0">{{ card.date_meta }}</p></div>
+                                        {% endif %}
```

Comment:
I think the padding `div`s would lead to a layout issue of only the date is shown, because we have that padding. 

Maybe we can simplify this by using the flex box `gap` property instead of using padding and wrappers?

Reply:
Chris has a ticket to tidy this up because my FE is terrible